src/python/bezier/_speedup.pyx

```diff
@@ -48,6 +49,9 @@ from bezier._triangle_intersection cimport CurvedPolygonSegment
 from bezier._triangle_intersection cimport TriangleContained
 
 
+np.import_array()
```

Comment:
Shot in the dark based on:

- https://numpy.org/doc/stable/numpy_2_0_migration_guide.html#functionality-moved-to-headers-requiring-import-array
- https://github.com/cython/cython/wiki/tutorials-numpy#c-api-initialization